mountpoint-s3-fs/tests/common/fuse.rs

```diff
@@ -299,7 +299,7 @@ pub mod mock_session {
         }
     }
 
-    fn create_test_client(client: Arc<MockClient>, prefix: &str) -> impl TestClient {
+    fn create_test_client(client: Arc<MockClient>, prefix: &str) -> impl TestClient + use<> {
```

Comment:
What does `use<>` mean?

Reply:
It means that the `impl TestClient` that we return does not capture any generic parameters (the lifetime of the `prefix` reference in this case). See [reference on precise capturing](https://doc.rust-lang.org/reference/types/impl-trait.html#r-type.impl-trait.generic-capture.precise).